lib/domain/model/preview_file/supported_preview_file_types.dart

```diff
@@ -1,3 +1,13 @@
+enum SupportedPreviewFileTypesEnum {
+  image,
+  doc,
+  excel,
+  powerPoint,
```

Comment:
why it is supported? and are you sure it is supported in all platform?

Reply:
I based myself on 'iOSSupportedTypes', 'androidSupportedTypes' but you are right I should only leave pdf and image for now

Reply:
yes, that the point now, but the name of it is not clear enough, so people confuse. The name is `Support to preview` but in the details we can not.